### DISCUSSION TITLE
The data in the events table continues to grow, leading to disk overload

### DISCUSSION THREAD

**Discussion by Jengro777** [getredash/redash#6803] [Q&A]
I have configured regular cleaning, but the data in the events table of the database continues to increase and will not decrease regularly

```
#/opt/redash/.env内容
PYTHONUNBUFFERED=0
#日志级别
REDASH_LOG_LEVEL=INFO
#redis连接字符串
REDASH_REDIS_URL=redis://redis:6379/0
# REDASH_REDIS_URL=redis://:123456@127.0.0.1:6379/0
#用于session加密
REDASH_COOKIE_SECRET=***
#用于数据库字段值加密，例如数据源账户密码
REDASH_SECRET_KEY=***

#数据库连接字符串，格式为postgresql://{db-user}:{db-password}@{db-ip}/{db-name}，根据需求自行更改
#postgres
POSTGRES_PASSWORD=***
REDASH_DATABASE_URL=postgresql://postgres:***@postgres/postgres


#开启自定义代码
REDASH_FEATURE_ALLOW_CUSTOM_JS_VISUALIZATIONS="true"
#开启特殊数据源
REDASH_ADDITIONAL_QUERY_RUNNERS="redash.query_runner.python"
# REDASH_ADDITIONAL_QUERY_RUNNERS="redash.query_runner.oracle,redash.query_runner.python"



#定期清理
REDASH_QUERY_RESULTS_CLEANUP_ENABLED="true"
REDASH_QUERY_RESULTS_CLEANUP_COUNT="10000"
REDASH_QUERY_RESULTS_CLEANUP_MAX_AGE="1"
```

**Comment by rei-ber** [comment]
Same problem here. Would be nice to have the possibility to remove the events automatically like query results. Is there any plan to implement it? A workaround could be `DELETE FROM events WHERE created_at < CURRENT_DATE - 7;` to delete events older than 8 days. 8, because `created_at` contains minutes, hours and seconds as well.

**Comment by gaecoli** [comment]
At present, the Redash jast have clean query_result job, you can delete events table manually or write a job for clean events job.
